src/app/lib/atlasmap-data-mapper/services/mapping.serializer.ts

```diff
@@ -306,6 +307,9 @@ export class MappingSerializer {
         }
         for (let fieldMapping of json.AtlasMapping.mappings.mapping) {
             var m: MappingModel = new MappingModel();
+            if (fieldMapping.id) {
+                m.uuid = fieldMapping.id;
+            }
             m.fieldMappings = [];
 
```

Comment:
The use of `var` is discouraged in TypeScript applications. The convention is to replace its use completely by using either `let` whenever the variable is meant be reassigned or `const` to signal that such variable is immutable by reference. Also, naming something like `m` is not a good practice. Something like this would work better and ease future maintainability by other peer developers: 

```
for (let fieldMapping of json.AtlasMapping.mappings.mapping) {
   // Please note below that it is not necessary to annotate types in assignation expressions
  const mappingModel = new MappingModel();

  if (fieldMapping.id) {
     mappingModel.uuid = fieldMapping.id;
  }

  mappingModel.fieldMappings = [];
```

Keeping object instance naming consistent with its associated type allows for easer and quicker refactoring in the future, and a more intuitive debugging when necessary.